components/SidebarNav/style.module.scss

```diff
@@ -23,6 +23,7 @@ $submenu-width: 180px;
 .sidebar {
   position: fixed;
   top: 0;
+  left: 0;
```

Comment:
Can we remove this unrelated change pls
```suggestion
```

Reply:
It is needed. If there is padding in the wrapper (aka Storybook), if `left` isn't specified, it'll be lined up with the padding, creating a little gap.

Reply:
I see. Was that meant to be in this PR? https://github.com/masonitedoors/MDL/pull/102
This PR was just adding a variant to the OverflowMenu component.

Reply:
yea

Reply:
I think it's fine here just wanted to call it out 👍this looks good though.